src/main/java/org/cactoos/text/Concatenated.java

```diff
@@ -43,6 +44,19 @@ public Concatenated(final Text... txts) {
         this(new IterableOf<Text>(txts));
     }
 
+    /**
+     * Ctor.
+     * @param strs Strings to be concatenated
+     */
+    public Concatenated(final String... strs) {
+        this(
+            new Mapped<>(
+                TextOf::new,
+                new IterableOf<String>(strs)
```

Comment:
@kokodyn Do you need explicit type parameter here?

Reply:
@andreoss fixed